account.go

```diff
@@ -0,0 +1,47 @@
+package quickbooks
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type AccountQueryResponse struct {
+	Accounts []*Account `json:"Account"`
+}
+
+type Account struct {
+	ID      string `json:"Id"`
+	Name    string `json:"Name"`
+	Active  bool   `json:"Active"`
+	Type    string `json:"AccountType"`
+}
+
+type AccountFilter struct {
+	IsActive bool
+	Type     string
+}
+
+func (a *AccountFilter) Eq() string {
+	sqlSelect := "SELECT * FROM Account"
+	sqlIsActive := "WHERE Active = " + strconv.FormatBool(a.IsActive)
```

Comment:
I guess it doesn't really matter for now since, for our use case, we'll always be providing both parameters, but I think you should really be writing this such that it will only conditionally add filters based on whether a value is populated in the filter struct. Like if the `Type` string is empty, you shouldn't try to filter by type.